pkg/kubelet/server/stats/summary.go

```diff
@@ -242,23 +243,112 @@ func (sb *summaryBuilder) latestContainerStats(info *cadvisorapiv2.ContainerInfo
 	return latest, true
 }
 
+// hasMemoryAndCPUInstUsage returns true if the specified container info has
+// both non-zero CPU instantaneous usage and non-zero memory RSS usage, and
+// false otherwise.
+func hasMemoryAndCPUInstUsage(info *cadvisorapiv2.ContainerInfo) bool {
+	if !info.Spec.HasCpu || !info.Spec.HasMemory {
+		return false
+	}
+	cstat, found := latestContainerStats(info)
+	if !found {
+		return false
+	}
+	if cstat.CpuInst == nil {
+		return false
+	}
+	return cstat.CpuInst.Usage.Total != 0 && cstat.Memory.RSS != 0
```

Comment:
We check `info.Spec.HasCpu` and `info.Spec.HasMemory` first (as in `containerInfoV2ToStats`). If `info.Spec.HasMemory` is true `cstat.Memory` will be non-nil. If `info.Spec.HasCpu` is true `cstat.Cpu` will be non-nil, but `cstat.CpuInst` may be nil.